### ISSUE TITLE
(Greater) consolidation of search results

### ISSUE THREAD

**Issue by DarrenPIngram** [Prowlarr/Prowlarr#190]
It would be good if the search results could be better consolidated to individual titles, as is standard with NZBHydra.

For example searching for "my feet are killing me" brings up many references, as expected, but they are not consolidated.

You would expect them to be consolidated by at least the file name, e.g. 
My.Feet.Are.Killing.Me-First.Steps.S01E01.Achy.Break-y.Feet.1080p.WEB.h264-B2B

As it is presently, you get the same file name/title replicated per each indexer it has been found within.

**Comment by austinwbest** [comment]
So ive never used hydra, how do you see these consolidated ones? If a user has 5 indexers and wants to pick from one but they are all consolidated, how is that accomplished there?

Some kind of accordion or something so you click on it to expand all the same releases from different sites i guess

**Comment by DarrenPIngram** [comment]
You are correct. I made a search for ‘Clarkson’ and the second entry from the top had a + symbol that I checked. Here you can view different versions/trackers. See https://ibb.co/fd75GbB

Otherwise you could select the main entry to download etc. See https://ibb.co/jTXHPLy

I hope this helps highlight what I would like to see implemented.




—- 
Via iPad. Hopefully with not too many typos.
